packages/contracts-bedrock/src/L2/L1BlockHolocene.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { L1BlockInterop, ConfigType } from "src/L2/L1BlockInterop.sol";
+import { StaticConfig } from "src/libraries/StaticConfig.sol";
+import "src/libraries/L1BlockErrors.sol";
+
+/// @custom:proxied
+/// @custom:predeploy 0x4200000000000000000000000000000000000015
+/// @title L1BlockHolocene
+/// @notice Interop extenstions of L1Block.
+contract L1BlockHolocene is L1BlockInterop {
+    /// @notice Event emitted when a new batcher hash is set.
+    event BatcherHashSet(bytes32 indexed batcherHash);
+
+    /// @notice Event emitted when a new ecotone gas config is set.
+    event GasConfigEcotoneSet(uint32 indexed blobBaseFeeScalar, uint32 indexed baseFeeScalar);
```

Comment:
replace w/ BasefeeScalarSet and BlobBasefeeScalarSet

Reply:
I changed the naming convention to the idiomatic casings for the new code, but didn't change the existing naming convention in the old code, e.g., https://github.com/ethereum-optimism/optimism/blob/a36291dc345382094ba971ffd4cd5987e84b989f/packages/contracts-bedrock/src/L2/L1Block.sol#L41 and https://github.com/ethereum-optimism/optimism/blob/a36291dc345382094ba971ffd4cd5987e84b989f/packages/contracts-bedrock/src/L2/L1Block.sol#L44, since it may break things. We can do it in the future.